rdr_service/services/consent/files.py

```diff
@@ -200,35 +206,51 @@ def _get_source_prefix(self) -> str:
 
 class CeConsentFactory(ConsentFileAbstractFactory):
     def _is_primary_consent(self, blob_wrapper: '_ConsentBlobWrapper') -> bool:
-        pass
+        pdf = blob_wrapper.get_parsed_pdf()
+        return pdf.has_text([(
```

Comment:
Similar to previous ask, can we refer to the expected version strings at the top of the PDF that we expect to have the matching text for each of these `has_text` calls?